docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+
+## Creating an Office Add-in 
+
+You can create an Office Add-in by using the Yeoman generator for Office Add-ins or Visual Studio.
+
+[!include[Yeoman vs Visual Studio comparision](../includes/yeoman-generator-recommendation.md)]
+
+### Yeoman generator for Office Add-ins
+
+The [Yeoman generator for Office Add-ins](https://github.com/officedev/generator-office) can be used to create a Node.js Office Add-in project that can be managed with Visual Studio Code or any other editor. The generator can create Office Add-ins for any of the following:
+
+- Excel
+- OneNote
+- Outlook
+- PowerPoint
+- Project
+- Word
+- Excel custom functions
+
+You can choose to create the project using HTML, CSS and JavaScript, or using Angular or React, and can choose between JavaScript and Typescript as well.
+
+Each [5-minute quick start](../index.md) contains step-by-step instructions that describe how to create an Office Add-in using the Yeoman generator. Within a quick start, select the `Yeoman generator` tab to display the proper set of instructions, as shown in the following screenshot.
+
+![A screenshot that shows quick start instructions with the Yeoman generator tab selected](../images/quick-start-tab-yeoman.png)
+
+### Visual Studio
+
+Visual Studio can be used to create Office Add-ins for Excel, Word, PowerPoint, or Outlook. An Office Add-in project gets created as part of a Visual Studio solution, which means you can use Visual Studio features like selecting **Start** or choosing **F5** to automatically run your add-in locally on IIS. Office Add-in projects that you create with Visual Studio use HTML, CSS, and JavaScript. Visual Studio can create Office Add-ins for any of the following:
+
+- Excel
+- Outlook
+- PowerPoint
+- Word
+
+The [5-minute quick start](../index.md) for each of these four applications contains step-by-step instructions that describe how to create an Office Add-in using Visual Studio. Within a quick start, select the `Visual Studio` tab to display the proper set of instructions, as shown in the following screenshot.
+
+![A screenshot that shows quick start instructions with the Visual Studio tab selected](../images/quick-start-tab-vs.png)
+
+## Exploring APIs with Script Lab
+
+Script Lab is an add-in that enables you to explore the Office JavaScript API and run code snippets while you're working in an Office program such as Excel or Word. It's available for free via [AppSource](https://appsource.microsoft.com/product/office/WA104380862) and is a useful tool to include in your development toolkit as you prototype and verify the functionality you want in your add-in. In Script Lab, you can access a library of built-in samples to quickly try out APIs or even use a sample as the starting point for your own code. 
```

Comment:
Is Outlook the only host without Script Lab?

Reply:
Script Lab only supports Excel, Word, and PowerPoint. So the hosts that don't support it are: Outlook, OneNote, Project. 

I intentionally avoided listing specifically which hosts support it, because that list will change over time and I'd rather not have to have to keep this doc in sync as that happens -- instead, I link to the Script Lab listing in AppSource, and to the main article about Script Lab, both of which will surely be updated as support for additional hosts is introduced.